### ISSUE TITLE
Python Migrator script to migrate docker images from ECR to Artifactory docker repo is not working on Python version 3

### ISSUE THREAD

**Issue by harishankaradsejfrog** [jfrog/docker2artifactory#26]
Description
Problem description: While trying to migrate the Docker repositories (container images) from Amazon Elastic Container Registry to the Artifactory docker repository using dockermigrator.py script, it fails when using python3 version. 
https://github.com/jfrog/docker2artifactory?tab=readme-ov-file#amazon-ecr-registry-migrator

What is the impact? 
Unable to use the script to do the migration of docker images from AWS ECR to JFrog Artifactory while using python3.

What is the expected behavior? 
Migration script should work on the latest version of python.

Steps to reproduce: 
- Push sample public docker images to AWS ECR.
- Install AWS CLI, Docker and Artifactory on test instance.
- Generate the AWS token and run the dockermigrator.py script to migrate the docker images from AWS ECR to JFrog Artifactory docker repo.

Possible workaround: NA
Environment: NA

**Comment by VibeDevOpsing** [comment]
Yes, as the solution you can use this my tool for AWS ECR https://github.com/anatolii-d/ecr2artifactory
